### ISSUE TITLE
Website runtime error: 8775869a

### ISSUE THREAD

**Issue by peej** [peej/to.uri.st#8]
Automatically reported error generated by the site at runtime:

```
offset may not be above 1000

  File "/base/python_lib/versions/1/google/appengine/ext/webapp/__init__.py", line 507, in __call__
handler.get(*groups)
  File "/base/data/home/apps/to-uri-st/1.340348580227241919/controllers/recent.py", line 58, in get
attractions = recent.fetch(26, (page - 1) * 26)
  File "/base/python_lib/versions/1/google/appengine/ext/db/__init__.py", line 1616, in fetch
raw = raw_query.Get(limit, offset, rpc=rpc)
  File "/base/python_lib/versions/1/google/appengine/api/datastore.py", line 1183, in Get
limit=limit, offset=offset, prefetch_count=limit, **kwargs)._Get(limit)
  File "/base/python_lib/versions/1/google/appengine/api/datastore.py", line 1113, in _Run
raise _ToDatastoreError(err)
```

**Comment by peej** [comment]
Automatically reported request data of erroring request:

```
GET /recent.html?page=2000
Accept: application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
Accept-Charset: ISO-8859-1,utf-8;q=0.7,*;q=0.3
Accept-Language: en-GB,en-US;q=0.8,en;q=0.6
Cookie: ****
Host: to-uri-st.appspot.com
User-Agent: Mozilla/5.0 (X11; U; Linux i686; en-US) AppleWebKit/532.9 (KHTML, like Gecko) Chrome/5.0.307.11 Safari/532.9,gzip(gfe)
```

**Comment by peej** [comment]
Added AppEngine BadRequestError to exception handler. Closed by 5ad8d4d730fe25aaa3d5e49c130be590cb2658b9
